cmd/wallet-js-sdk/test/specs/common.js

```diff
@@ -36,10 +36,18 @@ export function getJSONTestData(filename) {
 }
 
 // loadFrameworks loads agent instance
-export async function loadFrameworks({name = 'user-agent', logLevel = ''} = {}) {
+export async function loadFrameworks({
+      name = 'user-agent',
+      logLevel = '',
+      mediaTypeProfiles = ["didcomm/aip2;env=rfc19"],
```

Comment:
so by default aries framework loads with this media profile? even for DIDComm V1 tests?

Reply:
This media profile is for DIDComm V1, but I'll let baha confirm intention.

Reply:
this is for DIDComm V1 tests, yes.

DIDComm V2 have its own value